src/plugins/xfconf/README.md

```diff
@@ -0,0 +1,51 @@
+- infos = Information about the xfconf plugin is in keys below
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/xfconf
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained libc configurable experimental unfinished concept limited memleak
+- infos/metadata =
+- infos/description = storage plugin for xfconf
+
+## Introduction
+
+This is a storage plugin to mount the xfconf configuration settings.
```

Comment:
Please explain what this plugin can do, how to use it, what works, ... E.g. see src/plugins/augeas/README.md for a similar plugin.

Reply:
I added a much better introduction with 56e94e476fdc36ce01cfba6360e4a3e78c7c1bf5.